packages/client/hmi-client/src/views/Model.vue

```diff
@@ -114,30 +116,90 @@ onMounted(async () => {
 	);
 
 	// write json to model-json and draw model to model-drawn:
-	d3.select('#model-json').text(JSON.stringify(model));
 	await renderer?.setData(g);
 	await renderer?.render();
 });
+
+// TODO: let the user choose the model to display
+const selectedModelId = ref('1');
+
+const model = ref<Model | null>(null);
+// Whenever selectedModelId changes, fetch model with that ID
+watchEffect(async () => {
+	const result = await getModel(selectedModelId.value);
+	model.value = result as Model;
+});
```

Comment:
We have been down the road of tracing implicit dependencies, let's use `watch` instead.